### ISSUE TITLE
Title bar keeps apearing

### ISSUE THREAD

**Issue by patrickmast** [sindrets/atom-title-bar-replacer#29]
Hello,

Even when I set the UN-checkbox "Display title bar", the title bar keeps appearing on startup of Atom.
This is a short movie that shows the issue:
https://d.pr/v/uePjkn

Thanks!

**Comment by sindrets** [comment]
@patrickmast This is reproducible on my end as well. Thanks for reporting. I'll look into this.

**Comment by sindrets** [comment]
@patrickmast Meanwhile, if it is the title text that is bothering you, you can append this to your stylesheet to hide it:
```css
.title-bar-replacer .tbr-title-bar .custom-title {
    visibility: hidden;
}
```

**Comment by patrickmast** [comment]
Ok works great. Thanks!

**Comment by Scythern** [comment]
How are things going with this? I've started getting this bug too. When I open atom after a while the default title bar comes back.

**Comment by sindrets** [comment]
Fixed in v2.0.0.
